src/nupic/algorithms/SpatialPooler.hpp

```diff
@@ -84,7 +84,7 @@ namespace nupic {
                         Real minPctActiveDutyCycles=0.001,
                         UInt dutyCyclePeriod=1000,
                         Real maxBoost=10.0,
-                        Int seed=1,
+                        Int seed=-1,
```

Comment:
Oh I see that is intended. I'm not sure what we do elsewhere but we should decide whether seeds should default to fixed or random.